packages/dagu_car/src/kinematics_node.py

```diff
@@ -275,6 +283,59 @@ def trim(self, value, low, high):
 
         return max(min(value, high), low)
 
+    def check_and_adjust_radius(self, msg):
+        """
+        Checks if the current commands would generate a too small turning radius. If so it changes them.
```

Comment:
Can you explain exactly how they are changed? Also, please mention in the class description that such a procedure is applied.